src/lisp.h

```diff
@@ -1007,11 +1007,6 @@ INLINE bool
 #define FIXNUM_OVERFLOW_P(i) \
   (! ((0 <= (i) || MOST_NEGATIVE_FIXNUM <= (i)) && (i) <= MOST_POSITIVE_FIXNUM))
 
-INLINE ptrdiff_t
-clip_to_bounds (ptrdiff_t lower, EMACS_INT num, ptrdiff_t upper)
-{
-  return num < lower ? lower : num <= upper ? num : upper;
-}
```

Comment:
Please leave a function declaration here. This has added a new warning when compiling the C:

```c
  CC       xdisp.o

xdisp.c: In function ‘init_iterator’:

xdisp.c:2821:9: warning: implicit declaration of function ‘clip_to_bounds’ [-Wimplicit-function-declaration]

         (-1, XINT (BVAR (current_buffer, selective_display)),
```

Reply:
thanks, updated